graph/src/components/subgraph/instance.rs

```diff
@@ -10,7 +11,7 @@ pub struct DataSourceTemplateInfo {
 
 #[derive(Debug, Default)]
 pub struct BlockState {
-    pub entity_operations: Vec<EntityOperation>,
+    pub entity_cache: RefCell<EntityCache>,
```

Comment:
It's because of [this call](https://github.com/graphprotocol/graph-node/blob/master/runtime/wasm/src/module/mod.rs#L551-L555): `self.host_exports()` borrows self immutably, but we'd have to pass `&mut self.ctx` into `store_get`.

Reply:
I see, that code will work fine as:
```rust
        let entity = self.asc_get(entity_ptr);
        let id = self.asc_get(id_ptr);
        let entity_option = self
            .valid_module
            .host_exports
            .store_get(&mut self.ctx, entity, id)?;
```

Reply:
Aah, nice. Commit 9c6f39e8